projects/docs-tools/src/modules/code-examples/code-examples-editor.service.spec.ts

```diff
@@ -367,12 +367,11 @@ describe('Code examples editor service', () => {
       '@skyux/popovers': skyuxVersion,
       '@skyux/router': skyuxVersion,
       '@skyux/theme': skyuxVersion,
-      '@types/jasmine': '~4.3.1',
-      'ng2-dragula': '^5.0.1',
```

Comment:
I know we've had issue removing `ng2-dragula` in the past...

Reply:
I'm fairly certain it was an `ngcc` issue, which no longer applies. But we can save that for another release.